TSG/Update/Update-Hangs-after-Secret-Rotation.md

```diff
@@ -191,17 +191,6 @@ Function Restart-AzureLocalServices {
 
     $nodes = (Get-ClusterNode).Name
 
-    # Single Node Cluster
```

Comment:
This is not needed, this was an idea I was working on but went a different direction, forgot to remove.